src/goto-programs/remove_asm.cpp

```diff
@@ -107,15 +114,63 @@ void remove_asmt::gcc_asm_function_call(
     symbol.type=fkt_type;
     symbol.base_name=function_base_name;
     symbol.value=nil_exprt();
+    symbol.mode = ID_C;
```

Comment:
This is an important bugfix that may warrant its own commit (and regression test - @polgreen may have one).

Reply:
Moved to PR #2711